charts/zeebe-benchmark/templates/NOTES.txt

```diff
@@ -3,7 +3,7 @@
 Installed Zeebe cluster with:
 
  * {{  index .Values "camunda-platform" "core" "clusterSize" }} Brokers ({{ index .Values "camunda-platform" "core" "image" "repository" }}:{{ index .Values "camunda-platform" "core" "image" "tag" }})
-{{- if index .Values "camunda-platform" "operate" "enabled" }}
+{{if contains "operate" (index .Values "camunda-platform" "core" "env" 17 "value")}}
```

Comment:
I wanted to keep the image and taf in there, but that obviously doesn't matter since it's the same as the broker now that it's a single app 🤦 

I was quite proud of this solution though 😆